cmd/scheduler/app/options/options.go

```diff
@@ -141,6 +142,7 @@ func (s *ServerOption) AddFlags(fs *pflag.FlagSet) {
 		"Enable tracking of available storage capacity that CSI drivers provide; it is false by default")
 	fs.BoolVar(&s.EnableHealthz, "enable-healthz", false, "Enable the health check; it is false by default")
 	fs.BoolVar(&s.EnableMetrics, "enable-metrics", false, "Enable the metrics function; it is false by default")
+	fs.BoolVar(&s.EnablePprof, "enable-pprof", false, "Enable the pprof endpoint; it is false by default")
```

Comment:
Q: why not enable it by default, l remember k8s scheduler enabling pprof by default.

Reply:
This is the feedback from the security audit team, the original words were "The debug endpoint should be opt-in instead of opt-out," so it is turned off by default, and users can choose to turn it on